src/cf-typography/src/molecules/pull-quote.less

```diff
@@ -0,0 +1,26 @@
+.m-pull-quote {
+
+    &_body {
+        .heading-3();
+
+        color: @pull-quote;
+
+        .respond-to-max( @bp-xs-max, {
+            .heading-4();
+        } );
+    }
+
+    &_citation {
+        .heading-5();
+
+        color: @pull-quote_citation;
+    }
+
+    &__large .m-pull-quote_body {
+        .heading-2();
+
+        .respond-to-max( @bp-xs-max, {
+            .heading-3();
```

Comment:
Why didn't we use `h#()` mixins here, so we didn't have to write the extra media queries?